myproject/app/api/commant.py

```diff
@@ -7,52 +7,55 @@
 from flask_jwt_extended import JWTManager, create_access_token
 from app.models.model import db, user
 from flask_jwt_extended import get_jwt_identity, jwt_required
-
+from app.services.commant_services import target , comments
+from app.error_management.error_response import e_response , Response
+from app.error_management.success_response import success_response
+from app.validator.validators import check_command_required_fields
 
```

Comment:
make import in a sequence like
python modules
flask/Django modules
app  modules
put single line space between them